index.js

```diff
@@ -1,47 +1,89 @@
 'use strict';
 
-//setup requirements of express server
+//--SETUP AND INITIALIZATION
+//Setup requirements of express server
 require('dotenv').config();
 const cors = require('cors');
 const express = require('express');
 const app = express();
-
-//enable cors
+const axios = require('axios');
+//Enable cors
 app.use(cors());
-
-//define PORT based on environment
+//Define PORT based on environment
 const PORT = process.env.PORT;
 
-//server paths
+
+//--SERVER PATHS
 app.get('/hello', (req, res) => res.send('Hi there! :)'));
 app.get('/weather', serveWeather);
-app.get('/*', (req,res) => res.status(500).json({error: 'Server resource does not exist!'}));
+app.get('/movies', serveMovie);
+app.get('/*', (req, res) => res.status(500).json({ error: 'Server resource does not exist!' }));
+
 
-//server Data
-let weatherData = require('./weather.json');
+//--SERVER DATA HANDLING
+//Retrieve Weather data from weatherbit.io API
+async function retrieveWeather(lat, lon) {
+  return await axios.get(`https://api.weatherbit.io/v2.0/forecast/daily?key=${process.env.WEATHER_API_KEY}&lat=${lat}&lon=${lon}&days=7&units=I`);
+}
+
+//Retrieve Movie data from TMDB API
+async function retrieveMovies(place) {
+  return await axios.get(`https://api.themoviedb.org/3/search/movie?api_key=${process.env.MOVIE_API_KEY}&language=en-US&query=${place}&page=1&include_adult=false`);
+}
```

Comment:
Hi there!

I like your thought process here, separating out requests into separate functions so that you can reuse them :)
If you want, you can even move these into another file (for requests only etc).
Also, I would check out the axios instance as it seems like something you may like to use in future 👍

Reply:
I just looked up the axios instance and that's super cool! If I'm understanding it correctly, you can treat your newly assigned name as an axios object, and just pass the path you want into GET instead of the entire URI, right? If so that'd be super useful for API's that handle multiple paths. Thanks for the tip!

Reply:
Exactly! Yes it is awesome because you can set the instance in it's own file with all your other functions that make requests (which I think helps clean up code a bit)

Reply:
Oh! Also see axios interceptors! Also really neat :) (like middleware in a way)